### ISSUE TITLE
Is there any way to change the routing. ie: make i so <localhost>:<eelPort> redirects to main.html instead of having to do <localhost>:<eelPort>/main.html

### ISSUE THREAD

**Issue by bribrah** [samuelhwilliams/Eel#339]

**Comment by samuelhwilliams** [comment]
If you want this just for the home page, calling the file 'index.html' should suffice as this is generally the name of the default document returned by webservers. Is that what you mean/does that help?

**Comment by bribrah** [comment]
It does not, after changing the file to index.html I still get a "404 not found" when attempting to connect to "localhost:8000/" from chrome

**Comment by bribrah** [comment]
So I managed to implement this by making the following changes in eels __init__.py
```
BOTTLE_ROUTES = {
    "/eel.js": (_eel, dict()),
    "/<path:path>": (_static, dict()),
    "/eel": (_websocket, dict(apply=[wbs.websocket])),
    "/": (_static, dict())
}
```
```
def _static(path="index.html"):
```

**Comment by samuelhwilliams** [comment]
Hi @bribrah. I've done something very similar in https://github.com/samuelhwilliams/Eel/pull/340 that should meet your needs. 👍

**Comment by samuelhwilliams** [comment]
Released as v0.13.2
